feat: add support for 8275 chip and remove ament_cmake

canf-map opened the pull request:
- Implemented functions for 8275 chip.
- Remove the dependency on ament_cmake.
- Upgrade VERSION to 1.1.0 in CMakeLists.txt.

canf-map committed:
feat: add support for 8275 chip

- Implemented functions for 8275 chip.

build: update CMakeLists files and package.xml

- Remove the dependency on ament_cmake.
- Upgrade VERSION to 1.1.0 in CMakeLists.txt

Signed-off-by: Canfeng Zhuang <czhuang@qti.qualcomm.com>

canf-map committed:
fix: support all 8275 chip boards

- Fix bug to support all 8275 chip boards
- Upgrade VERSION to 1.1.1

Signed-off-by: Canfeng Zhuang <czhuang@qti.qualcomm.com>

jiaxshi commented on libqrc-udriver/package.xml:
```diff
@@ -7,9 +7,6 @@
   <license>BSD-3-Clause-Clear</license>
 
-  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+  <build_depend>gpiod</build_depend>
```
it should be libgpiod-dev

jiaxshi commented on libqrc/package.xml:
```diff
@@ -4,16 +4,9 @@
   <name>libqrc</name>
   <version>0.0.0</version>
```
please update the version and ensure that all versions are consistent.

jiaxshi commented on libqrc-udriver/src/qti_qrc_udriver.c:
```diff
@@ -43,7 +43,9 @@ int qrc_udriver_open(void)
             strcpy(QRC_FD, "/dev/ttyHS2");
-        } else if (strstr(buffer, "Addons IQ 9075 EVK") != NULL) {
+        } else if (strstr(buffer, "IQ 9075 EVK") != NULL) {
+            strcpy(QRC_FD, "/dev/ttyHS2");
```
I suggest using an array to map models to their serial names. This way, you can traverse the array without using multiple else-if statements.

canf-map committed:
fix: update code style, license headers, and ROS dependency, Config.cmake.in

- Apply clang-format for code style consistency
- Update license headers to comply with project requirements
- Correct ROS dependency in package.xml
- Add Config.cmake.in template

Signed-off-by: Canfeng Zhuang <czhuang@qti.qualcomm.com>

canf-map committed:
refactor: replace repetitive if-else model mapping with global struct array

- Introduced a global model_info_t array for model-to-tty/gpiochip/gpio mapping.
- Added find_model_info() helper function for unified lookup.
- Refactored qrc_udriver_open() and qrc_mcb_reset() to use the mapping table.
- Improved code maintainability and scalability for future model/device additions.

Signed-off-by: Canfeng Zhuang <czhuang@qti.qualcomm.com>

canf-map committed:
ci: bypass TinyFrame.h and TinyFrame.c from repolinter check

Signed-off-by: Canfeng Zhuang <czhuang@qti.qualcomm.com>

canf-map committed:
Merge branch 'qualcomm-qrb-ros:main' into main
